operator/src/main/java/oracle/kubernetes/operator/rest/RestServer.java

```diff
@@ -168,6 +185,18 @@ public void stop() {
     LOGGER.exiting();
   }
 
+  public String getInternalCertificate() {
```

Comment:
This is supposed to return base64 encoded pem - right?
Might want to rename it to match.
Add javadoc and LOGGER.entering ... since this is a public method (i.e. the RESTServer traces all public methods).
Please add javadoc & LOGGING for the new static create/destroy methods too.